ts/parts/Interaction.ts

```diff
@@ -1525,14 +1526,31 @@ extend(Series.prototype, /** @lends Highcharts.Series.prototype */ {
         // Don't loop over points on a series that doesn't apply inactive state
         // to siblings markers (e.g. line, column)
         if (inherit && inactiveOtherPoints && series.points) {
-            series.points.forEach(function (point: Highcharts.Point): void {
-                if (point.setState) {
-                    point.setState(state);
-                }
-            });
+            series.setAllPointsState(state);
         }
     },
 
+    /**
+     * Set the state for all points in the series.
+     *
+     * @function Highcharts.Series#setAllPointsState
+     *
+     * @private
+     *
+     * @param {string} [state]
+     *        Can be either `hover` or undefined to set to normal state.
+     */
+    setAllPointsState: function (
```

Comment:
Do we really need to add this function to the series prototype? I feel that it is already quite bloated with utility functions.
Since this is used only in one place, at least for now, it could be an internal utility function in the same file.

Reply:
Yes, we need this here. It's used in two files (`Pointer.js` and `Series.js`) and it's a very common task for series with `inactiveOtherPoints=true` (pie, sankey, networkgraph etc.) - it's called on every point/node/link/slice mouseOver/Out.

IMHO private functions are bad in a library like Highcharts - no way to wrap them, making customizations, plugins and test hard/impossible to write. There's no reason for protected/private functions in our case - we gain nothing, giving up flexibility.

Alternatively, we can create `Series.prototype.utils` object, like in `NavigationBindings` class. Maybe we should make this a common practice?

Reply:
> Yes, we need this here. It's used in two files (Pointer.js and Series.js) and it's a very common task for series

Okay, then it makes sense to leave it as is.